content/docs/command-reference/remote/modify.md

```diff
@@ -344,7 +344,7 @@ application. To use a custom authentication method, use the following parameters
 (listed in order of precedence):
 
 1. `connection_string` is used for authentication if given (all others params
-   are ignored).
+   are ignored / should be removed from the config).
```

Comment:
No need to add that phrase I think. It's possible you want to save the other config params there for later use, for example (removing `connection_string`).

Reply:
This should probably read "all others auth params except `account_name` are ignored", though...

Reply:
I just double checked. IMHO even `account_name` is ignored when the `connection_string` parameter is set (as it includes the account_name)

Reply:
Good point. I clarified in f1513021.